core/domain/suggestion_services.py

```diff
@@ -588,38 +588,48 @@ def get_all_suggestions_that_can_be_reviewed_by_user(user_id):
     ])
 
 
-def get_reviewable_suggestions(user_id, suggestion_type):
+def get_reviewable_suggestions(user_id, suggestion_type, limit, offset):
     """Returns a list of suggestions of given suggestion_type which the user
     can review.
 
     Args:
         user_id: str. The ID of the user.
         suggestion_type: str. The type of the suggestion.
+        limit: int. The maximum number of results to return.
+        offset: int. The number of results to skip.
 
     Returns:
-        list(Suggestion). A list of suggestions which the given user is allowed
-        to review.
+        Tuple of (results, next_offset). Where:
+            results: list(SuggestionModel). A list of suggestions that are
```

Comment:
We should not be returning models in the domain layer, can we convert them to domain objects instead?